.github/workflows/weeklyInteraction.yml

```diff
@@ -0,0 +1,22 @@
+name: "Weekly Interaction"
+on: 
+  schedule:
+    - cron: "0 15 * 1-4 0"
+    - cron: "0 15 * 9-12 0"
```

Comment:
This sets it so that the GitHub Action will only run 
**every Sunday at 15:00 UTC (aka 10:00 am EST) from January to April and September to December**

cron jobs run in UTC and is 4/5 hrs ahead of EST depending on daylight savings

Reply:
I _think_ you could combine both of these statements together

<img width="1332" alt="image" src="https://user-images.githubusercontent.com/59672089/211229185-a682308d-dfb6-4f2f-acb6-0c9ce92dc803.png">

```suggestion
    - cron: "0 15 * 1-4,9-12 0"
```

Reply:
yep, that should be work the same. i was copying an example from a doc